### ISSUE TITLE
Issue with integrating html-bundler-webpack-plugin with React in a multi-page setup

### ISSUE THREAD

**Issue by hqw567** [webdiscus/html-bundler-webpack-plugin#21]
Hello @webdiscus , I hope you're doing well. I have an issue regarding the usage of the html-bundler-webpack-plugin with React in a multi-page setup. I'm currently working on a project and I want to integrate this plugin with React, but I'm encountering some difficulties. I'm unsure about the proper setup and configuration to ensure seamless compatibility between the plugin and React. Could you please provide some guidance or instructions on how to effectively use the html-bundler-webpack-plugin with React in a multi-page setup? It would be great if you could share any insights, best practices, or examples to help me resolve this issue. Thank you for your time and assistance. I look forward to your response!

**Comment by webdiscus** [comment]
Hi @hqw567,

the very BIG problem with React project, what in all React project the style files are imported in JavaScript:
```js
import React from 'react';
import ReactDOM from 'react-dom';

import './style.scss'; // <= WTF ??, this is the wrong place to bind style to html
...
```

This is the `BAD PRACTICE`, this is the `wrong way` which is popularize with `mini-css-extract-plugin` and `html-webpack-plugin` :-(

Therefore I have created the pug-plugin and the html-bundler-webpack-plugin to build web site with the `right way` - specify style and script source files directly in HTML template.

Unfortunately, many UI npm packages imports styles in JS.

 I am currently working on the next version to add compatibility with projects where is used this "wrong way".
After release `v2.0` I can help you to set up a multipage React project.

Can you please create a small repo with your simple multipage React app? I'm not an expert in React.

**Comment by webdiscus** [comment]
Hi @hqw567,

I have released new `v2.0.0` version with support for React projects.
Please try new version.

**Comment by hqw567** [comment]
Hi @webdiscus ,

Thank you so much for your response! I truly appreciate your support and assistance. I'm really excited about the new v2 version with React support, and I believe it will bring a better experience and functionality to my project.

Thanks again!
